communities/microgalaxy/lab/sections/1_data_import_and_preparation.yml

```diff
@@ -60,6 +71,31 @@ tabs:
               - fastq
               - fasta
         button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkraken2%2Fkraken2"
+
+  - id: interactive-tools
+    title: Interactive tools
+    heading_md: >
+      Interactive tools for microbial research 
+    content:
+      - title_md: <code>Interactive JupyterLab Notebook</code>
+        description_md: >
+          <p>
+            The Jupyter Notebook is an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text. Uses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.
+          </p>
+        inputs:
+          - datatypes:
+              - Jupyter Notebook
+        button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_jupyter_notebook"
+      - title_md: <code>Pavian</code> - Interactive analysis of metagenomics data
+        description_md: >
+          <p>
+            An interactive browser application for analyzing and visualization metagenomics classification results from classifiers such as Kraken, KrakenUniq, Kraken 2, Centrifuge and MetaPhlAn. Pavian also provides an alignment viewer for validation of matches to a particular genome.
+          </p>
+        inputs:
+          - datatypes:
+              - Kraken report
+              - MetaPhlAn report
+        button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_pavian"
```

Comment:
I would add:
- RStudio
- Phinch
- Phyloseq
- Apollo

Reply:
Added all except apollo i dont know also which one to add for it